ui/package.json

```diff
@@ -6,7 +6,9 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.2.2",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.5.0",
```

Comment:
It's ok to use it for this task, but let's not use any UI library moving on

Reply:
Also, it's better to install exact dependencies (with `--save-exact` or `-E`) because it guarantees version equality between environments. I personally have the experience from a real project where "floating" dependencies broke the production pipeline

Reply:
Ok